deltachat-ios/View/TextFieldCell.swift

```diff
@@ -121,7 +201,72 @@ class TextFieldCell: UITableViewCell {
         // see: https://stackoverflow.com/a/36365399
         // therefore we use .default to capitalize the first character of the name
         cell.textField.keyboardType = .default
-        cell.textField.delegate = delegate
+        cell.textFieldDelegate = delegate
         return cell
     }
 }
+
+extension TextFieldCell: UITextFieldDelegate {
+
+    func textFieldDidBeginEditing(_ textField: UITextField) {
+        if let delegate = textFieldDelegate {
+            delegate.textFieldDidBeginEditing?(textField)
+        }
+    }
+
+    func textFieldDidEndEditing(_ textField: UITextField) {
+        if let delegate = textFieldDelegate {
+            delegate.textFieldDidEndEditing?(textField)
+        }
+    }
+
+    func textFieldDidChangeSelection(_ textField: UITextField) {
+        if #available(iOS 13.0, *), let delegate = textFieldDelegate {
+            delegate.textFieldDidChangeSelection?(textField)
+        }
+    }
+
+    func textFieldShouldClear(_ textField: UITextField) -> Bool {
+        if let delegate = textFieldDelegate, let result = delegate.textFieldShouldClear?(textField) {
+            return result
+        }
+        return true
+    }
+
+    func textFieldShouldReturn(_ textField: UITextField) -> Bool {
+        if let delegate = textFieldDelegate, let result = delegate.textFieldShouldReturn?(textField) {
+            return result
+        }
+        return true
+    }
+
+    func textFieldShouldBeginEditing(_ textField: UITextField) -> Bool {
+        preferValue = true
```

Comment:
if the text field was focussed it gains preference over the title until it is unfocussed again.
This way it is easier to read the whole value string while typing.